src/app/app.component.html

```diff
@@ -22,7 +22,23 @@
     </mat-form-field>
   </div>
 
-  <div class="dataDiv">
-    <app-weather-data *ngIf=" locationID || locationID == 0 " [data]="weatherData[locationID]"></app-weather-data>
+  <div class="centerPage">
+    <ng-container *ngIf=" locationID || locationID == 0">
+      <div class="timeBtns">
+        <b>Select a Time</b>
+        <ng-container *ngFor=" let time of weatherData[locationID].timeSeries; index as i ">
+          <div *ngIf="i<=24">
+            <ng-container
+              *ngIf="(weatherData[locationID].timeSeries[0].time | date:'d') == (weatherData[locationID].timeSeries[i].time | date:'d')">
+              <button (click)=" timeID = i ">{{weatherData[locationID].timeSeries[i].time | date:'shortTime'}}</button>
```

Comment:
You've got a reference to `weatherData[locationID].timeSeries[i]` already as time within the loop (the `let time of weatherDa....`. bit creates this)
```suggestion
              <button (click)=" timeID = i ">{{time.time | date:'shortTime'}}</button>
```

might want to call it timestep though so it's `timestep.time` instead of `time.time` 😆